### ISSUE TITLE
Icongenie does not add name="android" and name="iOS" to platform tags

### ISSUE THREAD

**Issue by customautosys** [quasarframework/quasar#12948]
### What happened?

After generating icons, the tags added to config.xml for cordova are only `<platform>` rather than `<platform name="android">` or `<platform name="ios">`

### What did you expect to happen?

Platform names are added

### Reproduction URL

https://github.com/quasarframework/quasar/tree/dev/icongenie

### How to reproduce?

Try it with a fresh project
npm init quasar
quasar mode add cordova
icongenie g -i ...

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Icon Genie CLI (@quasar/icongenie)

### Platforms/Browsers

Android

### Quasar info output

```Shell
Operating System - Linux(5.13.0-35-generic) - linux/x64
NodeJs - 16.14.0

Global packages
  NPM - 8.5.5
  yarn - Not installed
  @quasar/cli - 1.3.2
  @quasar/icongenie - 2.5.1
  cordova - 11.0.0

Important local packages
  quasar - 2.6.1 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.0.0-beta.8 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.13.4 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.31 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.14
  pinia - 2.0.12 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  vite - 2.8.6 -- Native-ESM powered web dev build tool
  eslint - Not installed
  electron - 17.2.0 -- Build cross platform desktop apps with JavaScript, HTML, and CSS
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - walfin-mini
  wlx3420035321cf - 192.168.1.182
```

**Comment by rstoenescu** [comment]
I've tested it and it works fine?

The only thing that I can think of is that you haven't yet added the Cordova iOS or Android platform.
"quasar mode add cordova" just creates the `src-cordova` folder, but it doesn't yet add iOS or Android to it. This comes after a quasar dev/build command where you also need to specify the platform (eg. quasar dev -m cordova -T ios).

Please feel free to comment or reopen if I missed anything.

**Comment by rstoenescu** [comment]
Also, make sure that you're not running multiple "quasar dev"s simultaneously. Or at all. Issue the icongenie command when no "quasar dev" command is running.

**Comment by rstoenescu** [comment]
Other than this, I tested this multiple times and really nothing changed in a long time on Icongenie that could introduce bugs :/

**Comment by customautosys** [comment]
For real it had the same problem again. This time I explicitly did a cordova platform add android before running icongenie.

![bildo](https://user-images.githubusercontent.com/14951259/160625180-64fbac00-c470-49ef-a5e7-eea75f03fb2f.png)

Not sure if this is a Quasar CLI Vite specific problem or whether it is to do with the options I passed in: --theme-color 1D1D1D --skip-trim

Was not running dev either.

Also if I run it twice, it will create another set of 2 platform elements (also without name="android" and name="ios")

**Comment by rstoenescu** [comment]
Will test this one more time, but not very confident I can reproduce what you're saying :(

**Comment by janjas92** [comment]
I was the same problem. I was created two example projects (first on vite cli and second on webpack). On this two projects i had the same problem . So to resolve this i added param name="android" to param platform
![obraz](https://user-images.githubusercontent.com/16246374/161445516-69aee6b6-0e98-4298-8172-b90da46da157.png)

```
Requirements check results for android:
Java JDK: installed 1.8.0
Android SDK: installed true
Android target: installed android-Tiramisu,android-32,android-31,android-30,android-29,android-28,android-27,android-26,android-25,android-24,android-23
Gradle: installed /usr/share/gradle/bin/gradle

```

```
------------------------------------------------------------
Gradle 4.4.1
------------------------------------------------------------

Build time:   2012-12-21 00:00:00 UTC
Revision:     none

Groovy:       2.4.17
Ant:          Apache Ant(TM) version 1.10.7 compiled on October 24 2019
JVM:          1.8.0_221 (Oracle Corporation 25.221-b11)
OS:           Linux 5.4.0-107-generic amd64

```

**Comment by rstoenescu** [comment]
Was finally able to reproduce it. Really weird....
Fix available in `@quasar/icongenie` v2.5.4

**Comment by customautosys** [comment]
> Was finally able to reproduce it. Really weird.... Fix available in `@quasar/icongenie` v2.5.4

Thank you Razvan!
